lib/v2/gcores/collection.js

```diff
@@ -0,0 +1,130 @@
+        })
+        .get();
+
+    if (list.length > 0 && list.every((item) => item.url === undefined)) {
+        throw new Error('Article URL not found! Please submit an issue on GitHub.');
+    }
+
+    const out = await Promise.all(
+        list.map((item) => {
+            const articleUrl = `https://www.gcores.com${item.url}`;
+            return ctx.cache.tryGet(articleUrl, async () => {
+                const itemRes = await got({
+                    method: 'get',
+                    url: articleUrl,
+                });
+
+                const itemPage = itemRes.data;
+                const $ = cheerio.load(itemPage);
+
+                let articleData = await got(`https://www.gcores.com/gapi/v1${item.url}?include=media`);
+                articleData = articleData.data.data;
+                let cover;
+                if (articleData.attributes.cover) {
+                    cover = `<img src="https://image.gcores.com/${articleData.attributes.cover}" />`;
+                } else if (articleData.attributes.thumb) {
+                    cover = `<img src="https://image.gcores.com/${articleData.attributes.thumb}" />`;
+                } else {
+                    cover = '';
+                }
+
+                const articleContent = JSON.parse(articleData.attributes.content);
+                const entityRangeMap = {};
+                for (const block of articleContent.blocks || []) {
+                    if (block.entityRanges.length) {
+                        entityRangeMap[block.key] = block.entityRanges;
+                    }
+                }
+
+                $('figure').each((i, elem) => {
+                    const keyAttr = elem.attribs['data-offset-key'];
+                    const keyMatch = /^(\w+)-(\d+)-(\d)$/.exec(keyAttr);
+                    let actualContent = '';
+                    if (keyMatch) {
+                        const [, key, index] = keyMatch;
+                        if (entityRangeMap[key] && entityRangeMap[key][index]) {
+                            const entityKey = entityRangeMap[key] && entityRangeMap[key][index].key;
+                            const entity = articleContent.entityMap[entityKey];
+                            actualContent = convertEntityToContent(entity);
+                        }
+                    }
+
+                    if (actualContent) {
+                        $(elem).replaceWith(actualContent);
+                    }
+                });
+
+                // remove editor toolbar img
+                $('.md-editor-toolbar').replaceWith('');
+                // remove hidden tip block
+                $('.story_hidden').replaceWith('');
+
+                const content = $('.story.story-show').html();
+                return {
+                    title: item.title,
+                    description: cover + content,
+                    link: articleUrl,
+                    guid: articleUrl,
+                };
+            });
+        })
+    );
```

Comment:
Please use the API of `/gapi/v1/collections/${collection}` and `/gapi/v1/collections/${collection}/originals` instead. They include both the feed metadata and full articles.